packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -98,4 +100,31 @@ describe('quantic-tab', () => {
       Expect.activeTabContains(tabs.knowledge.label);
     });
   });
+
+  describe('when using accessibility', () => {
+    beforeEach(() => {
+      loadFromUrlHash({}, `tab=${tabs.all.label}`);
+    });
+
+    it('when SpaceKey is pressed', () => {
+      TabSelectors.button().first().focus().type(' ');
+      Expect.activeTabContains(tabs.all.label);
+    });
+
+    it('when TabKey && SpaceKey is pressed', () => {
+      TabSelectors.button().first().focus().tab().type(' ');
```

Comment:
I'm curious why we have to add `type('  ')` here ?

Reply:
type('  ') will trigger a space key press, simulating the situation where users use keyboard to trigger the button.